debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/YBVerifyRecord.java

```diff
@@ -27,4 +27,9 @@ public static void isValidUpdate(SourceRecord record, String pkField, int pk) {
     hasValidKey(record, pkField, pk);
     isValidUpdate(record, true);
   }
+
+  public static void isValidDelete(SourceRecord record, String pkField, int pk) {
```

Comment:
don't see this getting used? Is it called by by parent class?

Reply:
There was an existing test which had the call to `YBVerifyRecord#isValidDelete` but the test was disabled, once enabled it started failing as the method originally pointed to `VerifyRecord#isValidDelete` because it was not defined in our derived class.

So this method was added to fix the test failure.